doc/changelog.md

```diff
@@ -42,6 +42,12 @@
 
 <h3>Improvements</h3>
 
+* Catalyst gradient functions ``grad``, ``jacobian``, ``jvp``, and ``vjp`` can now be invoked from
+  outside a ``@qjit`` context. This simplifies the process of writing functions where compilation
+  can be turned on and off easily by adding or removing the decorator. The functions dispatch to
+  their JAX equivalents when the compilation is turned off.
+  [(#375)](https://github.com/PennyLaneAI/catalyst/pull/375)
```

Comment:
```suggestion
* Catalyst gradient functions `grad`, `jacobian`, `jvp`, and `vjp` can now be
  invoked from outside a `@qjit` context. This simplifies the process of
  writing functions where compilation can be turned on and off easily by adding
  or removing the decorator. The functions dispatch to
  their JAX equivalents when the compilation is turned off.
  [(#375)](https://github.com/PennyLaneAI/catalyst/pull/375)
```

Reply:
Quick side question, do you enforce a very narrow line width Josh (our standard is 100)?

Reply:
Haha true, what is the GitHub comment line length, 80?

Reply:
```suggestion
* Catalyst gradient functions `grad`, `jacobian`, `jvp`, and `vjp` can now be invoked from
  outside a `@qjit` context. This simplifies the process of writing functions where compilation
  can be turned on and off easily by adding or removing the decorator. The functions dispatch to
  their JAX equivalents when the compilation is turned off.
  [(#375)](https://github.com/PennyLaneAI/catalyst/pull/375)
```